packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -63,7 +63,11 @@ export default {
 	},
 	computed: {
 		popperClass() {
-			return this.size ? `kt-popper kt-popper--${this.size}` : `kt-popper`
+			const classes = ['kt-popper', `kt-popper-${this.size}`]
```

Comment:
why single-hyphen the size; it's a modification

Reply:
It’s a typo

Reply:
css also needs editing then

Reply:
@carsoli CSS is fine as far as I can tell

Reply:
ok so ur classes are not on this pattern `kt-popper--size-sm`

Reply:
so just missing the `--size` here

Reply:
@carsoli Are you absolutely sure you saw the nested `&--size`?

Reply:
Okay, I see why you’re confused now I think, the CSS is fine the code still isn’t